parsers/verilog.c

```diff
@@ -1577,13 +1580,32 @@ static int tagNameList (tokenInfo* token, int c)
 	{
 		c = processType(token, c, &kind);	// update token and kind
 
-		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`')
+		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`' )
 		{
 			if (kind != K_UNDEFINED && kind != K_IDENTIFIER)	// ignore procedual assignment: foo = bar;
 				createTag (token, kind);
 			if (c == '=')
 				c = skipExpression (c);
 		}
+		else if (c == '(' || c == '[') // should be instance? TODO: function void foo();
+		{
+			c = skipDimension(c);
+			if(c == '(')
+			{
+				c = skipPastMatch ("()");
+				/* c= skipWhite (c); */
+				if(c == ';' || c == ',')
+				{
+					kind = K_INSTANCE;
```

Comment:
This variable is not necessary.

Reply:
OK, I'm sorry for my poor coding skill 😭